notification_lib/novu_manager/subscribers_manager.py

```diff
@@ -164,3 +164,37 @@ def delete_subscriber(cls, subscriber_id: str) -> None:
         )
 
         cls.handle_response(response, "Can't delete subscriber !").json()
+
+    @classmethod
+    def update_subscriber_credentials(
+        cls, subscriber_id: str, provider: str, credentials: SubscriberCredentials
```

Comment:
provider_id should be more suitable then provider since you are using it as an id no ?

Reply:
novu end consider it id, but in our end we consider it provider name